int32 timedist
float64 urate
float64 initFeedbackStrength
int32 regularization
float64 synboost
float64 maxSpeed
float64 epsh
float64 pretension
float64 maxForce
float64 springMult1
float64 springMult2
int32 diff
float64 amplitude
int32 delay
int32 guideType
int32 guideIdx
float64 guideAmpl
float64 guideFreq
bool learning
int32 targetForce
int32 range